README.md

```diff
@@ -389,6 +390,34 @@ Executive Trusonafications require the user to scan an identity document to auth
 
 [^1]: You must provide at least one field that would allow Trusona to determine which user to authenticate. The identifier fields are `DeviceIdentifier`, `TruCode`, `UserIdentifier`, and `EmailAddress`.
 
+
+
+### Canceling a Trusonafication
+
+Any Trusonafication that is `IN_PROGRESS` can be cancelled.
+
+```csharp
+var trusona = new Trusona(
+  token: "token",
+  secret: "secret"
+);
+
+var request = Trusonafication.Essential()
+                             .UserIdentifier("73CC202D-F866-4C72-9B43-9FCF5AF149BD")
+                             .Action("login")
+                             .Resource("Acme Bank")
+                             .Build();
+
+var trusonafication = await trusona.CreateTrusonafication(request);
+
+// if still IN_PROGRESS after some predetermined time, you may cancel it
+
+await trusona.CancelTrusonafication(trusonafication.id);
+```
+
+If the trusonafication cannot be successfully cancelled based on its status at the point the request is received, or if the trusonafication does not exist, an exception will be thrown.
+
+
 ### Using TruCode for device discovery
 
 In the previous section, we demonstrated how to issue a Trusonafication to a specific device using it's `DeviceIdentifier`, but what if the user is trying to login to your website from their desktop computer and you don't know what the user's `DeviceIdentifier` is? That's where TruCode comes in.
```

Comment:
The start of this paragraph is now wrong. It's reference the cancel section but talking about issuing a trusonafication